jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc

```diff
@@ -0,0 +1,809 @@
++
+NOTE: When you create your project, make sure you also enable API access. If you do not enable API access, the command we will run next (to create your cluster) will fail.
++
+Setting the default `project id`, compute `zone` and `region` will make running subsequent commands easier:
++
+[source,shell]
+----
+gcloud config set project <PROJECT_ID>
+gcloud config set compute/zone <ZONE>
+gcloud config set compute/region <REGION>
+----
+. Create your cluster 
++
+TIP: CircleCI recommends using https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[workload identities] to allow workloads/pods in your GKE clusters to impersonate Identity and Access Management (IAM) service accounts to access Google Cloud services. Use the following command to provision a simple cluster:
++
+[source,shell]
+----
+gcloud container clusters create circleci-server \
+  --num-nodes 3 \
+  --machine-type n1-standard-4 \
+  --workload-pool=<PROJECT_ID>.svc.id.goog
+----
++
+NOTE: Your kube-context should get updated with the new cluster credentials automatically.
++
+If you need to update your kube-context manually, you can by running the following:
++
+[source,shell]
+----
+gcloud container clusters get-credentials circleci-server
+----
+. Install the https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke[GKE authentication plugin] for `kubectl`:
++
+[source,shell]
+----
+gcloud components install gke-gcloud-auth-plugin
+----
+. Verify your cluster:
++
+[source,shell]
+----
+kubectl cluster-info
+----
+. Create a service account:
++
+[source,shell]
+----
+gcloud iam service-accounts create <SERVICE_ACCOUNT_ID> --description="<DESCRIPTION>" \
+  --display-name="<DISPLAY_NAME>"
+----
+. Retrieve credentials for the service account:
++
+[source,shell]
+----
+gcloud iam service-accounts keys create <KEY_FILE> \
+  --iam-account <SERVICE_ACCOUNT_ID>@<PROJECT_ID>.iam.gserviceaccount.com
+----
+endif::env-aws[]
+
+[#enable-workload-identities-in-gke]
+==== Enable Workload Identities in GKE (optional)
+Follow these steps if you already have a GKE cluster and need to enable workload identities on the cluster and the node pools.
+
+. Enable Workload Identity on existing cluster:
++
+[source,shell]
+----
+  gcloud container clusters update "<CLUSTER_NAME>" \
+    --workload-pool="<PROJECT_ID>.svc.id.goog"
+----
+. Get node pools of existing GKE cluster:
++
+[source,shell]
+----
+  gcloud container node-pools list --cluster "<CLUSTER_NAME>"
+----
+
+. Update existing node pools:
++
+[source,shell]
+----
+  gcloud container node-pools update "<NODEPOOL_NAME>" \
+    --cluster="<CLUSTER_NAME>" \
+    --workload-metadata="GKE_METADATA"
+----
+
+You must repeat Step 3 for all the existing node pools. Follow these links for steps to enable Workload Identities for your Kubernetes service accounts: 
```

Comment:
I don't think sending people off to these other docs is a good idea during the prerequs section. We need to just provide the steps again